turbiniactl

```diff
@@ -44,23 +44,42 @@ log = logging.getLogger('turbinia')
 logger.setup()
 
 
-def PrintTaskStatus(project, region, days=1, all_fields=False):
+def PrintTaskStatus(instance, project, region, days=0, task_id=None,
+                    request_id=None, all_fields=False):
   """Prints the recent history for Turbinia Tasks.
 
   Args:
+    instance (string): The Turbinia instance name.
```

Comment:
Maybe mention that this is the Pubsub queue name

Reply:
Done.